doc/ug/getting_started_verilator.md

```diff
@@ -25,13 +25,14 @@ There are 3 memory types: ROM, RAM and Flash.
 By default, the system will first execute out of ROM and then jump to flash.
 A program needs to be built for each until ROM functionality for code download is ready.
 
-For that purpose compile the demo program with "simulation" settings, which adjusts the frequencies to better match the simulation speed. 
+For that purpose compile the demo program with "simulation" settings, which adjusts the frequencies to better match the simulation speed.
 For more information on building software targets refer to the [Software Getting Started Guide]({{< relref "getting_started_sw.md" >}}).
 
 ```console
 $ cd $REPO_TOP
 $ ./meson_init.sh
-$ ninja -C build-out all
+$ ninja -C build-out sw/device/boot_rom/boot_rom_export_sim_verilator
+$ ninja -C build-out sw/device/examples/hello_world/hello_world_export_sim_verilator
```

Comment:
That change is not related to the PR description, please remove it or place it in a separate commit.

Apart from that, I'm not sure I like this change: the old command was easy to remember, the new command is almost impossible to remember. It doesn't matter if it builds slightly too much.

Reply:
It also builds quite quickly, so I think I'm good leaving this as the simpler command for the sake of readability.